django/db/models/sql/compiler.py

```diff
@@ -938,26 +938,28 @@ def get_converters(self, expressions):
                     converters[i] = (convs, expression)
         return converters
 
-    def apply_converters(self, row, converters):
-        row = list(row)
-        for pos, (convs, expression) in converters.items():
-            value = row[pos]
-            for converter in convs:
-                value = converter(value, expression, self.connection)
-            row[pos] = value
-        return tuple(row)
+    def apply_converters(self, rows, converters):
```

Comment:
Ahh missed that, I thought you suggested adding a new method.

> I guess that change doesn't make much sense given apply_converters() is called in some other places.

agreed, it would also be odd for the method to require a connection kwarg when it has one defined on `self`.